### DISCUSSION TITLE
Image is not added

### DISCUSSION THREAD

**Discussion by rabol** [lsnepomuceno/laravel-a1-pdf-sign#91] [Q&A] [answered]
Hi

Using the latest version of this package I sign and add the standard image to a pdf, but the image itself is not added to the pdf

Here is my code:
```
  // FROM FILE
  $cert = new ManageCert;
  $cert->setPreservePfx();
  $cert->fromPfx(storage_path('test.pfx'), '123456');

  // GENERATING IMAGE FROM CERTIFICATE
  $image = SealImage::fromCert($cert);

  // IMAGE STORAGE LOCATION
  $imagePath = a1TempDir(true, '.png');
  File::put($imagePath, $image);

  // Returning signed resource string
  $pdf = new SignaturePdf(storage_path('test.pdf'), $cert);
  $resource = $pdf->setImage($imagePath) // USE THE "setImage" METHOD
      ->signature();
  File::put(storage_path('test.pdf'),$resource);
```

I can find the image in a1TempDir, but it is not in the pdf

Any hint ?

Thanks
Kind regards
Steen

**Comment by rabol** [comment]
Small note: If I do this:

```setSealImgOnEveryPages()```
I get an image on all pages

**Comment by lsnepomuceno** [comment]
Hello @rabol,

I applied the correction and sent more details to the [ISSUE 92](https://github.com/lsnepomuceno/laravel-a1-pdf-sign/issues/92).

**Comment by lsnepomuceno** [comment, marked as answer]
Released version [v1.0.8](https://github.com/lsnepomuceno/laravel-a1-pdf-sign/releases/tag/1.0.8) with the correction of the problem.
